draft-wang-ppm-differential-privacy.md

```diff
@@ -239,11 +254,128 @@ The following considerations are out-of-scope for this document:
 
 This document uses the same conventions for error handling as {{!DAP}}.
 
-> TODO: add more
+Let `exp(EPSILON)` denote raising the numeric constant `e` to the power of
```

Comment:
Did you know that you can do this now:

```suggestion
Let `exp(ε)` denote raising the numeric constant `e` to the power of
```

Of course, whether you want to or not is something you will need to decide.

Reply:
What would the RFC editor make of this? It's massively preferable obviously, but I don't know if this would eventually get shot down.

Reply:
Well, they recently changed the tooling because the tooling was too strict.  This would be fine.

Reply:
It looks like it only renders in "art mode": 
![image](https://github.com/wangshan/draft-wang-ppm-differential-privacy/assets/3453007/070f9a8e-7fa9-46dc-a505-1cbbed32f6f2)

Reply:
srsly?  That's an xml2rfc bug if ever I saw one.  `&#120656;`= &#120656; works fine in browsers, so something is double-quoting the ampersand.

Reply:
FWIW, my local testing doesn't produce that bug.  I had no trouble getting an epsilon to show, either by pasting the character directly in the document source or by using an entity reference.

Reply:
Ack, I'll do some research at some point. Possibly I have an old version.